index.js

```diff
@@ -120,11 +120,7 @@ module.exports = Object.assign({}, Addon, {
     contents = JSON.parse(contents)
 
     // Add `ember-engines` to devDependencies by default
-    contents.devDependencies['ember-engines'] = '^0.8.5';
-
-    // Move `ember-cli-htmlbars` into dependencies from devDependencies
-    contents.dependencies['ember-cli-htmlbars'] = contents.devDependencies['ember-cli-htmlbars'];
```

Comment:
Since [the addon blueprint already adds `ember-cli-htmlbars` to `dependencies`](https://github.com/ember-cli/ember-cli/blob/master/blueprints/addon/index.js#L56-L60), this line was actually removing it again.